app/controllers/idv/confirmations_controller.rb

```diff
@@ -22,8 +22,15 @@ def submit_answers
       agent = Proofer::Agent.new(vendor: idv_vendor, applicant: idv_applicant)
       @idv_vendor = idv_vendor
       @confirmation = agent.submit_answers(idv_resolution.questions, idv_resolution.session_id)
-      complete_idv_profile if @confirmation.success?
+      successful_confirmation if @confirmation.success?
       clear_idv_session
     end
+
+    def successful_confirmation
```

Comment:
Assuming you mean proofing/verification was successful why not use one of those words?  Introducing a third word for the same thing (`confirmation`) seems like a bad idea.